aea/package_manager/v1.py

```diff
@@ -304,11 +312,11 @@ def json(
     @classmethod
     def from_dir(
         cls,
-        packages_dir: Path,
+        packages_dir: Union[Path, str],
```

Comment:
We should strictly use `pathlib.Path` objects for paths imo.

Reply:
@angrybayblade no problem here, but it makes usage a bit easier in case string provided.
can clean it if needed

Reply:
done